server/meson_options.txt

```diff
@@ -0,0 +1,26 @@
+option('dbus_service_dir',
+  type: 'string',
+  description: 'Directory for D-Bus service files'
+)
+
+option('systemd',
+  type: 'feature',
+  value: 'auto',
+  description: 'Enable systemd support'
+)
+
+option('systemduserunitdir',
+  type: 'string',
+  description: 'Directory for systemd user service files'
+)
+
+option (
+  'profile',
+  type: 'combo',
+  choices: [
+    'default',
+    'development'
+  ],
+  value: 'default',
+  description: 'The build profile for Obfuscate. One of "default" or "development".'
```

Comment:
is it for obfuscate?

Reply:
Is it possible that i used the same copy paste inthe portal too? Lol

Reply:
seems to be!